pmm-admin.go

```diff
@@ -1216,7 +1216,7 @@ despite PMM server is alive or not.
 	flagMongoURI, flagCluster, flagDSN, flagFormat string
 	flagATags                                      string
 
-	flagVersion, flagJson, flagAll, flagForce bool
+	flagVersion, flagJson, flagAll, flagForce, flagDisableSSL bool
```

Comment:
var flagJson should be flagJSON

Reply:
So placed flagDisableSSL variable only after "flagForce".
Any problem? :-)

Reply:
I've got it.
You only say about the variable name.
But it's same as "flagMongoURI" parameter above.
And parameter name is "--disable-ssl", so this name is better. :-)
Thanks.